patches/tModLoader/Terraria/ModLoader/ModItem.cs

```diff
@@ -102,22 +100,24 @@ public class ModItem:ModTexturedType
 		/// </summary>
 		/// <param name="item">The item being cloned</param>
 		/// <param name="itemClone">The new item</param>
-		public virtual ModItem Clone(Item item) => NewInstance(item);
+		public virtual ModItem Clone(Item item) {
+			ModItem clone = Clone();
+			clone.item = item;
+			return clone;
+		}
```

Comment:
This looks like a correct change. However I'd like you to replace `Clone()` there with `MemberwiseClone` and remove the no-args clone method (it's not needed)

Reply:
Yeah, I was wondering why both methods exist. Will Do!